sympy/printing/latex.py

```diff
@@ -166,6 +166,7 @@ class LatexPrinter(Printer):
         "parenthesize_super": True,
         "min": None,
         "max": None,
+        "truncate": True,
```

Comment:
Using `truncate=False` expresses what is wanted in a negated sense. Maybe what would be better is the opposite like `full_output=True` (default is `False`) or something like that...